geometry_msgs/PointStamped obj_centroid
geometry_msgs/PointStamped placement
int32 action_type
float64 fraction